### PR TITLE
CP-41651 dynamic configuration copy right & product name

### PR DESCRIPTION
**Pull request by LunfanZhang**
Replace copy right, product name, release time with variables to enable dynamic configuration.

Signed-off-by: Lunfan Zhang luzhan@tibco.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Introduction
 
-This is the golang guest utilities for XenServer
+This is the golang guest utilities for {{PRODUCTNAME}}
```
Comment (xihuan-citrix) [inline README.md:5]: Not sure, but maybe change to "guest on Xen Hypervisor" is a better option

File: mk/xe-linux-distribution [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #! /bin/sh
 
-# Copyright (c) 2022, Cloud Software Group Holdings, Inc.
+# Copyright (c) {{TIME}}, {{COPYRIGHT}}
```
Comment (xihuan-citrix) [inline mk/xe-linux-distribution:3]: TIME seems not an accurate variable name, what about `COPYRIGHTTIME` and `COPYRIGHTAUTHOR`

File: README.md (hunk 1 above)
Comment (LunfanZhang) [reply README.md:5]: as a variable name. what a mouthful^_^

File: mk/xe-linux-distribution (hunk 2 above)
Comment (LunfanZhang) [reply mk/xe-linux-distribution:3]: Errorless

File: README.md (hunk 1 above)
Comment (LunfanZhang) [reply README.md:5]: change to "Xenxerver VM Tool" on next/ "Citrix Hypervisor VM Tool" on Yangtze

Comment (xihuan-citrix) [reply README.md:5]: Hi @LunfanZhang , this should be changed to `This is {{PRODUCTNAME}} implemented with golang`

**Review by xihuan-citrix — APPROVED** [review]

**Review by liulinC — APPROVED** [review]

**Pull request closed without merging**